tool.go

```diff
@@ -498,7 +498,21 @@ func main() {
 	cmdServe.Run = func(cmd *cobra.Command, args []string) {
 		options.BuildTags = strings.Fields(*tags)
 		dirs := append(filepath.SplitList(build.Default.GOPATH), build.Default.GOROOT)
-		sourceFiles := http.FileServer(serveCommandFileSystem{options: options, dirs: dirs, sourceMaps: make(map[string][]byte)})
+		root := ""
```

Comment:
@neelance, you're right, it's actually import for it to be the zero value.

I misread the code below, thinking it sets `root` in all cases, but it only sets it when `len(args) == 1`.

Given that's the case (that value of `root` must be zero value for string), I think both styles are okay.